topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -0,0 +1,690 @@
+---
+layout: tutorial_hands_on
+title: Downstream single-cell RNA analysis with RaceID
+zenodo_link: 'https://zenodo.org/record/1511582'
+tags:
+  - single-cell
+questions:
+  - What is normalisation and why is it necessary?
+  - How many types of unwanted variation are there?
+  - How are biological phenotypes clustered?
+  - What is the difference between PCA and tSNE?
+  - What is the difference between cell trajectory and cell fate?
+objectives:
+  - Filtering, normalising, and clustering cells in a matrix
+  - Assessing the quality of individual clusters
+  - Inferring a lineage between cell types
+  - Examining gene expression
+  - Determining the top most expressive genes per cluster
+  - Correcting for unwanted variation
+requirements:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+##        - scrna-introduction
+        - scrna-plates-batches-barcodes
+        - scrna-umis
+        - scrna_preprocessing
+
+time_estimation: 2H
+key_points:
+  - Clustering single-cell RNA-seq data is often noisy
+  - Understanding the heirarchal relationships between clusters provides an understanding of cell development 
```

Comment:
heirarchal  -> hierarchal

StemID can be used for ...
FateID can be used for ....?